scripts/ci-deploy.sh

```diff
@@ -56,11 +56,10 @@ for module in "${MODULES[@]}"; do
     npm --no-color run cdk diff -- --context env=${ENV}
     npm --no-color run cdk deploy -- --all --require-approval=never --context env=${ENV}
   else
-    pip --quiet --disable-pip-version-check --no-color install --upgrade -r "$dir"/../misc/requirements/requirements.txt
-
-    # install the collection for CI/CD
-    ansible-galaxy collection install --force git+https://github.com/ringier-data/ops-ci-aws.git,"${OPS_CI_AWS_BRANCH}"
-    ansible-galaxy collection install --upgrade ansible.posix community.aws amazon.aws community.general community.postgresql kubernetes.core
+    if [[ OPS_CI_AWS_BRANCH != "main" ]]; then
+      # install the collection for CI/CD (the main branch is already baked into the CodeBuild custom image)
+      ansible-galaxy collection install --force git+https://github.com/ringier-data/ops-ci-aws.git,"${OPS_CI_AWS_BRANCH}"
+    fi
```

Comment:
Please review/confirm the planned activities for the full migration:
Step 0, `ops-ci-aws`, `ops-ci-shared`, `ops-ci-codebuild`, `ops-ci-codebuild-image` have all relevant changes merged to the default branch. (almost done, except this repo itself)
Step 1, manual/local deployment into each AWS environment where CodeBuild plays the role (C+ x3, Iris x3, Alloy x4, SCMI x2, Svc x1)
Step 2, push a NO-CHANGE change to both `develop` and `main` of `ops-ci-codebuild`

Then the central deployment is done. For each repo, no change is technically necessary, but to benefit from the the financial as well as speed optimization:
Step 3, bump `ops-ci-shared` to the newest `main`
Step 4, adopt the Fargate task definition to use ARM instead of X86
Step 5, adopt `deploydest.yml` and/or the role parameters of `build_push_docker_image` to target ARM

Are we on the same page?

Reply:
>Step 1, manual/local deployment into each AWS environment where CodeBuild plays the role (C+ x3, Iris x3, Alloy x4, SCMI x2, Svc x1)

Yes, deployment of `ops-ci-codebuild` i assume? Before this i would do step (3-5) for `ops-ci-codbuild` (i.e. not consider it optional)

>Step 2, push a NO-CHANGE change to both develop and main of ops-ci-codebuild

What is purpose of this one? If (1) already did deploy `ops-ci-codbuild` everywhere - smoke test?

>Then the central deployment is done. For each repo, no change is technically necessary, but to benefit from the the financial as well as speed optimization: (...)

Agreed.

PS
https://github.com/ringier-data/ops-ci-aws/blob/develop/roles/build_push_docker_image/tasks/build-image.yml#L41 is currently failing builds for us (we work around this by setting `OPS_CI_AWS_BRANCH` = `1a7c823a41dd7dfb42498d7a5c5a4c68aaf5a1f0`). I think this line could be removed or moved into the `"use BuildKit if needed`" block